src/platform/utilities/sso/index.js

```diff
@@ -81,7 +81,12 @@ export async function checkAutoSession(
   profile = {},
 ) {
   const { ttl, transactionid, ...queryParams } = await ssoKeepAliveSession();
-
+  const removeReturnUrl = url => {
+    const isExternalCSP = sessionStorage
+      .getItem(url)
+      .includes('mhv' || 'myhealth' || 'ebenefits' || 'MAP');
```

Comment:
Change to `?.includes` just in case the `authReturnUrl` is null and let's only do `mhv` for now as I think some TeamSites are being impacted.